verification/VerifyTasks.wdl

```diff
@@ -21,7 +21,7 @@ task CompareVcfs {
   runtime {
     docker: "gcr.io/gcp-runtimes/ubuntu_16_0_4:latest"
     disks: "local-disk 50 HDD"
-    memory: "16 GiB"
```

Comment:
Could we consider using the memory-doubling retry strategy on this?

Reply:
Ultimately I want to deprecate this in favor of my GATK tool, so we don't need to spend too much time boyscouting.